force-app/main/default/profiles/IT Administrator.profile-meta.xml

```diff
@@ -48,6 +53,10 @@
     <layoutAssignments>
         <layout>Template__c-Template Layout</layout>
     </layoutAssignments>
+    <tabVisibilities>
```

Comment:
If you can delete this reference, it will cause problems with your deployment in the future.

Reply:
@LAWBowie Did you delete?